ground/build.gradle

```diff
@@ -267,7 +270,7 @@ dependencies {
     // WorkManager
     implementation "androidx.work:work-runtime:$workVersion"
     implementation "androidx.work:work-rxjava2:$workVersion"
-    androidTestImplementation "androidx.work:work-testing:$workVersion"
```

Comment:
Are we certain that this dependency is no longer needed? Could you please verify this by running the instrumentation tests locally?

Reply:
I'm still unable to run tests via AS; 0/0 run when initiated via the UI. I also ran `gradlew connectedAndroidTest` which failed due to `com.android.tools.r8.internal.a: Cannot fit requested classes in a single dex file (# methods: 184732 > 65536)` during `:sharedTest:mergeExtDexDebugAndroidTest`.

Reply:
After using a newer SDK image, I am able to run tests, but they're already failing at HEAD https://github.com/google/ground-android/issues/1410

We should be able to delete this dependency since we don't have integration tests for workers atm.